haiku-games/bong/patches/bong-1.0.0.patchset

```diff
@@ -0,0 +1,22 @@
+From a1dfc8cafd22c65ddbceb6948554a4c416e0affb Mon Sep 17 00:00:00 2001
+From: tts2k <tranthaison2000@gmail.com>
+Date: Thu, 21 Dec 2017 05:26:23 +0000
+Subject: fix for haiku
```

Comment:
Could you try upstream your patch, since a couple of GCI mentors are members of HaikuArchives and can accept your changes.

Reply:
Will do, after i finish the recipe